### DISCUSSION TITLE
J401 carrier Board Flash JetPack | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1223] [Q&A]
# J401 carrier Board Flash JetPack | Seeed Studio Wiki

Flash JetPack to reComputer J4012 (J401 carrier board)

https://wiki.seeedstudio.com/reComputer_J4012_Flash_Jetpack/

**Comment by cgpadwick** [comment]
I found an issue with the flashing process hanging and the flash failing.  I captured the issue and the fix in this forum post: https://forum.seeedstudio.com/t/j401-board-flashing-error-and-fix/276887
